telemetry/constants.go

```diff
@@ -0,0 +1,27 @@
+package telemetry
+
+const (
+
+	// Metric Names
+	CNIExecutimeMetricStr  = "CNIExecutionTimeMs"
+	CNIAddTimeMetricStr    = "CNIAddTimeMs"
+	CNIDelTimeMetricStr    = "CNIDelTimeMs"
+	CNIUpdateTimeMetricStr = "CNIUpdateTimeMs"
+
+	// Dimension Names
+	ContextStr        = "Context"
+	SubContextStr     = "SubContext"
+	VMUptimeStr       = "VMUptime"
+	OperationTypeStr  = "OperationType"
+	VersionStr        = "Version"
+	StatusStr         = "Status"
+	CNIModeStr        = "CNIMode"
```

Comment:
what is CNI mode?

Reply:
CNI mode tells single tenancy or multitenancy